src/dlgRoomProperties.cpp

```diff
@@ -243,15 +243,13 @@ QStringList dlgRoomProperties::getComboBoxSymbolItems()
         while (itSymbolUsed.hasNext()) {
             itSymbolUsed.next();
             if (itSymbolUsed.value() == symbolCountsList.at(i)) {
-                displayStrings.append(qsl("%1 {%2:%3}")
-                    .arg(itSymbolUsed.key())
-                    /*:
-                    This text will be part of a list of room values shown, which will show the value
-                    itself, followed by the counted number of rooms with this very value like:
-                    grey {count:2} - so please translate like counted ammount, number of, etc.
-                    */
-                    .arg(tr("count"))
-                    .arg(QString::number(itSymbolUsed.value())));
+                /*:
+                Format for showing a room symbol with its usage count. %1 is the symbol itself (e.g., "★" or "!"),
+                %2 is the number of rooms using this symbol. Example output: "★ (count: 5)" or "! (count: 12)".
+                The word "count" and the format can be translated, but ensure the numbers remain clearly associated.
+                */
+                displayStrings.append(tr("%1 (count: %2)", "Format for room symbol with usage count")
```

Comment:
Second argument is obsolete here?
```suggestion
                displayStrings.append(tr("%1 (count: %2)")
```